specification/sql/resource-manager/Microsoft.Sql/preview/2023-02-01-preview/JobPrivateEndpoints.json

```diff
@@ -0,0 +1,329 @@
+{
```

Comment:
Please fix the linter issues: [https://github.com/Azure/azure-rest-api-specs/pull/24902/checks?check_run_id=15311430363](https://github.com/Azure/azure-rest-api-specs/pull/24902/checks?check_run_id=15311430363).

Reply:
I would like to request an exemption for the linter issues. They look to be the same as the parent resource Job Agent which got approved in this PR. 
https://github.com/Azure/azure-rest-api-specs/pull/24293

Reply:
We can consider approving other linter errors, however, suggesting to fix these two linter errors:
1. DefaultErrorResponseSchema: Please refer to error response schema (you can even refer from common types v1). Adding just description, while it is helpful, it doesn't tell the clients how to parse the error response.
2. ProvisioningStateSpecifiedForLROPut: Please add provisioningState to the ARM Resource definition, as for async APIs, it is required to send it for client to see whether the resource provisioning has completed or not.

Reply:
Thank you! 
1. Could you please point me to other Microsoft.Sql specs that have the correct error response schema? I couldn't tell the difference between mine and existing ones 
2. We return a location header and an async operation header that allows customers to see the provisioning state and get back the completed object. Is that sufficient?

Reply:
Hi @jacobopac I am adding approval for Lint errors, as author wants to keep the older behavior for newer API as well. However, even if there are some issues which exist in previous APIs, it is always good to fix the newly introduced APIs to keep evolving your RP better and compliant to ARM RPC.

Reply:
Hey @visingla-ms , thank you very much! Definitely, I'll keep this in mind while we iterate over the APIs.